changelogs/current.yaml

```diff
@@ -10,6 +10,14 @@ behavior_changes:
     is properly extracted. This changes the Prometheus name from
     envoy_listener_myprefix_downstream_cx_overflow{} to envoy_listener_downstream_cx_overflow{envoy_listener_address="myprefix"}.
     This does not affect the Prometheus name if stat_prefix is not set.
+- area: listener
+  change: |
+    Fixed metric tag extraction so that worker_id is properly extracted from the listener stats. This changes the Prometheus name from
+    envoy_listener_worker_1_downstream_cx_active{envoy_listener_address="0.0.0.0_10000"} to envoy_listener_downstream_cx_active{envoy_listener_address="0.0.0.0_10000", envoy_worker_id="worker_1"} .
+- area: server
+  change: |
+    Fixed metric tag extraction so that worker_id is properly extracted fromt the server stats. This changes the Prometheus name from
+    server_worker_1_watchdog_miss{} to server_watchdog_miss{envoy_worker_id="worker_01}.
```

Comment:
The names should be prefixed with `envoy_` I think

Reply:
you are right, validate it with curl